src/components/App/App.js

```diff
@@ -0,0 +1,15 @@
+import Navigation from '../../layout/navigation/Sidebar';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import AddMotorcycle from '../motorcycleForm/addMotorcycle';
+import './App.css';
```

Comment:
The file/folder structure of a basic react app has App.js and App.css at the same level as the index.js file. I suggest you follow that structure and keep the App.js and App.css files at their original level in the file structure. Thank you